requirements/doc.txt

```diff
@@ -1,18 +1,18 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
@DeimerM I would prefer doing this with the python version this workflow was already using. Any reason to downgrade it to python 3.8?

Reply:
No reason @magajh, I just updated the requirements with python 3.10 thanks a lot for your help